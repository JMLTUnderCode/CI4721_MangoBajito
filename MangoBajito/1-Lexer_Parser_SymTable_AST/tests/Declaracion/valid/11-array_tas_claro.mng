# Test 11: Tipo array tas_claro.
se_prende(){
	jeva n0: tas_claro = Sisa;
	culito n1: tas_claro = Nolsa;
	culito decl0 : tas_claro[3];
	jeva decl1 : tas_claro[5] = [Sisa, Sisa, Sisa, Sisa, Nolsa];
	jeva decl2 : tas_claro[2] = [n0, n1];
	decl0[0] = Nolsa;
	decl0[1] = n0;
	
};