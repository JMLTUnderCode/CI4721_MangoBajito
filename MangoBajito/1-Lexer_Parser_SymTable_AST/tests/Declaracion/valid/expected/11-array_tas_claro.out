| AcomÃ³date que te lo estan analizando d:-)
Error: ARRAY_LITERAL_SIZE_MISMATCH
  - Sendo peo en la linea 6, columna 49: Dale que te caben mÃ¡s! Te faltan cositas que meterte. SÃ³lo llevas '0' de '3'.
--> Print AST? (s/n): [1;36m[5m
               =======================================================
                                   AST Representation                 
               =======================================================
[0m
Programa
|--> Main
     |--> InstrucciÃ³n
          |--> decl0 | Category: DeclaraciÃ³n | Type: tas_claro | Kind: VARIABLE
          |    |--> Literal | Category: Array_Size | Type: mango | Value: 2
          |--> AsignaciÃ³n | Kind: =
               |--> decl1 | Category: DeclaraciÃ³n | Type: tas_claro | Kind: CONSTANTE
               |    |--> Literal | Category: Array_Size | Type: mango | Value: 3
               |--> Secuencia | Category: ExpresiÃ³n | Kind: ,
                    |--> Secuencia | Category: ExpresiÃ³n | Kind: ,
                    |    |--> Literal | Category: Bool | Type: tas_claro | Kind: Sisa | Value: 1
                    |    |--> Literal | Category: Bool | Type: tas_claro | Kind: Sisa | Value: 1
                    |--> Literal | Category: Bool | Type: tas_claro | Kind: Nolsa | Value: 0
[1;31m[5m
               =======================================================
                       ---->        Error Program        <----        
               =======================================================
[0m
| Hay un poquito de cÃ³digo en tu marihuana d:-]
